pom.xml

```diff
@@ -521,12 +521,8 @@
         <artifactId>maven-surefire-plugin</artifactId>
         <version>2.19.1</version>
         <configuration>
-          <excludes>
-            <exclude>org/folio/rest/api/*Test.java</exclude>
-          </excludes>
           <includes>
-            <include>org/folio/rest/api/StorageTestSuite.java</include>
-            <include>**/*Test.java</include>
+            <include>org/folio/AllTestSuite.java</include>
           </includes>
```

Comment:
The renames will make the reviews even harder, so I will take the additional PR approach.
One of my reasons for moving into the Suite is to make the transition from junit4 to junit5 more explicit.
Anything removed from the suite would be removed because it is migrated to junit5.

The file rename you suggest also addresses this goal so I have no objections to doing this.

Reply:
I agree with splitting this into a separate PR.

I think it's worth discussing what we do more after this is merged.

If the goal is to isolate the tests more because they use shared infrastructure and state and that reduces their stability, then we may want to move all of the tests that require this style of integration to maven style integration tests.